docs/source/advancedcommands.rst

```diff
@@ -1,34 +1,12 @@
 Advanced Commands
 *****************
 
-Feasibility Analysis
+Relaxation Analysis
```

Comment:
I think we should leave it as "Feasibility Analysis", not necessarily because it's better, but because it's standard in the field (see e.g. Boyd's GP tutorial paper section 3.1).

I'd also be supportive of "Primal feasibility analysis".